src/storage/src/hummock/sstable_store.rs

```diff
@@ -86,7 +85,7 @@ impl SstableStore {
         }
     }
 
-    pub async fn put(&self, sst: Sstable, data: Bytes, policy: CachePolicy) -> HummockResult<()> {
+    pub async fn put(&self, sst: Sstable, data: Vec<u8>, policy: CachePolicy) -> HummockResult<()> {
         self.put_sst_data(sst.id, data.clone()).await?;
```

Comment:
Please remove clone because clone `Vector` will allocate new memory space while clone Bytes only increases the reference count.

Reply:
Seems hard to do this if we want to later fill the cache.

Reply:
Oh I forgot here. Only length is used below.